src/runtime/proxy.ts

```diff
@@ -20,31 +119,50 @@ export type CallableRegistry<RegistryT> = {
 	};
 };
 
+/**
+ * 
+ * @param ctx The {@link Context} object to use to call the scripts in
+ * accessible modules
+ * @param map A {@link MapFrom} object that describes how to map
+ * `[module, script] pairs from a camelCase registry to a snake_case registry
+ * @returns A {@link CallableRegistry} object that implements the "syntax sugar"
+ * that used in the `ctx.modules.<script>.<name>()` pattern to call scripts
+ * without the `ctx.call` function.
+ */
 export function buildRegistryProxy<RegistryT, RegistryCamelT>(
```

Comment:
why can't this object be generated statically instead of dynamically returning proxies? i'd imagine that's faster for calling & doesn't have a lot of overhead with script startup times.